python/src/trezorlib/cli/crypto.py

```
@@ -70,15 +72,20 @@ def decrypt_keyvalue(
+    help="include node information in the output",
+)
 @with_client
 def batch_get_publickeys(
-    client: "TrezorClient", paths: Tuple[str], curve: str
-) -> List[bytes]:
+    client: "TrezorClient", paths: Tuple[str], curve: str, include_node: bool
+) -> EcdsaPublicKeys:
     """Batch get publicKeys by given paths.
     paths: we want to used for public keys, e.g. "m/44'/0'/0'" "m/44'/0'/0'/0/0'"... support up to 20 path
     """
     assert (len(paths) <= 20, "Support up to paths' size 20")
```

Ruff (0.11.9)

Assert test is a non-empty tuple, which is always `True`

(F631)